ui/pages/Project/components/ProjectOverview.jsx

```diff
@@ -128,7 +128,25 @@ const ProjectOverview = React.memo((
       ),
     }))]), [])
   if (!datasetSections.length) {
-    datasetSections.push({ title: 'Datasets', content: 'No Datasets Loaded', key: 'blank' })
+    datasetSections.push({
+      title: 'Datasets',
+      content: (
+        <div>
+          No Datasets Loaded <br />
+          <i>Where is my data?</i> <Popup
+            trigger={<HelpIcon />}
+            hoverable
+            content={
+              <div>
+                Loading data from AnVIL to seqr is a slow process, and generally takes a week.
+                If you have been waiting longer than this for your data, please reach
+                out to <a href="mailto:seqr@broadinstitute.org">seqr@broadinstitute.org</a>
```

Comment:
Should this message be shown only when AnVIL auth is enabled?

Reply:
hmm thinking about this we should actually only show it if that specific project has an anvil workspace associated with it. Thanks for catching this